lib/bambora/v1/reports/merchants.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+module Bambora
+  module V1
+    module Reports
+      class Merchants
+        def self.get(merchant_id, options = {})
+          credentials = options[:credentials]
+
+          raise ArgumentError, "#{self}.get takes an integer argument" unless merchant_id.is_a?(Integer)
```

Comment:
A little bit of added developer experience improvement to better guide the developer towards inputting the correct value type.